modules/lib/lib-content/src/test/resources/test/QueryContentHandlerTest_dsl_sort.js

```diff
@@ -0,0 +1,75 @@
+var assert = require('/lib/xp/testing.js');
+var content = require('/lib/xp/content.js');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8706041)
```suggestion
var content = require("/lib/xp/content.js");
```